### DISCUSSION TITLE
Username email generator: forward email not working - An error has occurred.  Failed to execute 'json' on 'Response': body stream already read

### DISCUSSION THREAD

**Discussion by PyroghostX** [dani-garcia/vaultwarden#4867] [Q&A] [answered]
See the screenshot, in bitwarden.com its working, and same thing in vaultwarden is not working, giving me this error:
An error has occurred.
Failed to execute 'json' on 'Response': body stream already read

It generates the email in forward email, but it doesn't make it back to the client (none of them, desktop/web/extension)
And it works perfectly fine on bitwardens web interface

![chrome_8lL4ZCdbKB](https://github.com/user-attachments/assets/8aa473cf-ff23-4c64-a11d-e8fcdb1af2d9)

**Comment by BlackDex** [comment]
This is a fully client side feature.
So probably not something we can fix. 

Check the developer console of the browser (F12) and see what happens.

**Comment by PyroghostX** [comment]
![chrome_lHTerK6ds1](https://github.com/user-attachments/assets/4e92b21d-5123-4b38-8ae7-99da6720b852)

console log shows the same error

So if its not an issue on vaultwardens end, would it be an issue on forward email side? should I talk with them?

**Comment by BlackDex** [comment, marked as answer]
Looks like a client issue: https://github.com/bitwarden/clients/issues/10091
Which seems to be resolved via https://github.com/bitwarden/clients/pull/10075 and is integrated into v2024.7.3, which we do not support (yet).

As this is a client issue I'm going to move this to the discussions page.
